torch/_export/converter.py

```diff
@@ -427,6 +427,41 @@ def convert_as_noop(self, node: torch._C.Node):
         output_name = node.output().debugName()
         self.name_to_node[output_name] = args[0]
 
+    def convert_profiler__record_function_enter_new(self, node: torch._C.Node):
+        target = torch.ops.profiler._record_function_enter_new
+        args = tuple(self.get_fx_value(input) for input in node.inputs())
+        fx_node = self.fx_graph.call_function(target, args)
+        output_name = node.output().debugName()
+        self.name_to_node[output_name] = fx_node
+
+    def convert_profiler__record_function_exit(self, node: torch._C.Node):
+        # _record_function_exit has side effect so we keep it in fx.graph
+        # currently, _record_function_enter_new and _record_function_exit are
+        # discarded during `retrace_as_exported_program`.
+        target = torch.ops.profiler._record_function_exit
+        args = tuple(self.get_fx_value(input) for input in node.inputs())
+        self.fx_graph.call_function(target, args)
+
+    def convert_aten___is__(self, node: torch._C.Node):
+        # export() currently specializes on the sample inputs for aten::__is__
+        left, right = tuple(self.get_fx_value(input) for input in node.inputs())
+        fx_node = self.fx_graph.call_function(operator.is_, (left, right))
+        output_name = node.output().debugName()
+        self.name_to_node[output_name] = fx_node
+
+    def convert_aten___isnot__(self, node: torch._C.Node):
+        # export() currently specializes on the sample inputs for aten::__isnot__
+        left, right = tuple(self.get_fx_value(input) for input in node.inputs())
+        fx_node = self.fx_graph.call_function(operator.is_not, (left, right))
+        output_name = node.output().debugName()
+        self.name_to_node[output_name] = fx_node
+
+    def convert_aten___not__(self, node: torch._C.Node):
```

Comment:
looks like a lot of these ops can be converted following the same pattern. 

we can have a dict for mapping
{"aten::__not__", operator.not_ }

and have a shared function for converting them.